### PR TITLE
Build improvements

### PR DESCRIPTION
**Pull request by caarlos0**
- Locked dependencies versions
- renamed main file to `main.go`
- added goreleaser to manage releases

FWIW, the release would look like this: https://github.com/caarlos0/openvpn_exporter/releases/tag/0.2.2

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- env:
+  - CGO_ENABLED=0
+  goarch:
+  - amd64
+  goos:
+  - linux
+  - darwin
+dockers:
+- image_templates:
+  - "caarlos0/openvpn-exporter:latest"
```
Comment (pieterlange) [inline .goreleaser.yml]: Remove references to personal repo

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,13 @@
+module github.com/caarlos0/openvpn_exporter
```
Comment (pieterlange) [inline go.mod]: And again :)

File: .goreleaser.yml (hunk 1 above)
Comment (caarlos0) [reply .goreleaser.yml]: Sorry, don’t know where you want to push the images to...  what’s the Docker hub account you want the images to be pushed to?

File: go.mod (hunk 2 above)
Comment (caarlos0) [reply go.mod]: Oops 😬

**Comment by caarlos0** [comment]
fixed the issues and also updated the README file :)

**Comment by BartVerc** [comment]
Thank you for your pull request! 
I am going to test this way of building and publish the images on our docker-hub.

**Pull request merged**